adaptive_scheduler/kernel/fullscheduler_ortoolkit.py

```diff
@@ -32,6 +32,8 @@
     'SCIP': pywraplp.Solver.SCIP_MIXED_INTEGER_PROGRAMMING
 }
 
+FALLBACK_ALGORITHM = ALGORITHMS['SCIP']
```

Comment:
I guess we could - I figured that the fallback would basically just be a fallback from a commercial solver to a free one, and SCIP was the best free one. But I'm sure there is some type of problem out there were SCIP is worse then CBC so I guess it could be configurable